pkg/kubehound/graph/edge/share_ps_namespace.go

```diff
@@ -38,14 +38,25 @@ func (e *SharePSNamespace) Name() string {
 	return "SharePSNamespace"
 }
 
+func (e *SharePSNamespace) AttckTechniqueID() AttckTechniqueID {
+	return AttckTechniqueEscapeToHost
+}
+
+func (e *SharePSNamespace) AttckTacticID() AttckTacticID {
+	return AttckTacticPrivilegeEscalation
+}
+
```

Comment:
This doesn't match what's in `docs/reference/attacks/SHARE_PS_NAMESPACE.md`, there the tactic in `NA` and the Technique is tainted shared content (T1080)